### PR TITLE
[READY]Makes fireball slower

### PR DESCRIPTION
**Pull request by silicons**
Fireball, with its 65 instant damage, 50% knockout chance, incendiary effects, and 3x3 AOE proximity detonation, and rapid cooldown, does not deserve being as fast as other projectiles. The main problem with other projectiles is they were too easily dodged. This is not easily dodgeable at all on the old values, and maybe even not when slower, as it is proximity-detonating, and nowadays, a fully aimable projectile. 
Also didn't someone once say that the knockout chance wasn't intentional?

Edit: My mistake, explosion damage is randomized, but still very high. It's 10 brute + 10 burn + randomized  explosion damage + fire if you get hit and end up on a fire tile. That's usually very, very high.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
i ded

**Comment by ChangelingRain** [comment]
Frankly I think the entire 'double the speed of projectiles' thing was absurd.

**Comment by kevinz000** [comment]
It's expecially absurd with AOE projectiles, in my opinion.. But yeah, now that I think about it, a 50% increase would of been better I guess

**Comment by kevinz000** [comment]
If you're going to downvote because of the change, please specify that you want only fireballs to be slower, or nothing to be slower.

File: code/modules/projectiles/projectile/magic.dm [hunk 1]
```diff
@@ -55,6 +56,9 @@
 	if(ismob(target)) //multiple flavors of pain
 		var/mob/living/M = target
 		M.take_overall_damage(0,10) //between this 10 burn, the 10 brute, the explosion brute, and the onfire burn, your at about 65 damage if you stop drop and roll immediately
+					//65 damage with a single shot of a spammable spell is rediculous with knockout + fire.
```
Comment (optimumtact) [inline code/modules/projectiles/projectile/magic.dm]: This isn't a chat room, remove this

**Comment by optimumtact** [comment]
Get good

File: code/modules/projectiles/projectile/magic.dm (hunk 1 above)
Comment (kevinz000) [inline code/modules/projectiles/projectile/magic.dm]: ok sir

**Comment by optimumtact** [comment]
Also it appears your beef is mostly with fireball, so I'd prefer you nerf that instead of applying a big salty nerf hammer to the entire spell range and leaving some of our projectiles arbitrarily slow for no apparent reason.

**Comment by kevinz000** [comment]
ok

**Comment by optimumtact** [comment]
I just don't know about this.

I'd prefer that we nerfed the actual spell rather than making all our projectiles move at different speeds

@tgstation/commit-access feel free to overrule me

**Comment by KorPhaeron** [comment]
I don't remember which PR or thread I said this in, but I saw a wizard cast fireball earlier and the guy 5-6 tiles away just blow up without even getting to see the sprite it hit so fast.

I think that's a general projectile thing though, and lzimann has a PR up for it, so we should try that first and revisit giving this a snowflake speed/other nerf later

**Pull request closed without merging**